py_mind_memo/graphics.py

```
@@ -230,16 +248,12 @@ def get_font_metrics(style):
                 line_w += f_obj.measure(txt)
-                # フォントの高さ（アセント+ディセント）をベースにする
                 line_max_h = max(line_max_h, f_obj.metrics("linespace"))
             
             max_w = max(max_w, line_w)
             total_h += (line_max_h if line_max_h > 0 else size + 10)
             
-        if is_icon:
-            result = max(100, max_w + img_w + ICON_PADDING * 2), max(35, total_h + 12)
-        else:
-            result = max(100, max(max_w + 20, img_w + 20)), max(35, total_h + 12 + img_h)
+        result = max(100, max(max_w + icon_w + 20, img_w + 20)), max(35, max(total_h, icon_h) + 12 + img_h)
```

Ruff (0.15.7)

[warning] Nested `max` calls can be flattened

Flatten nested `max` calls

(PLW3301)